commit b5569228dd32aa848d2c8eed7e3f50d3a8522b8d
Author: Pierre Ynard <linkfanel@yahoo.fr>
Date:   Sun Jan 23 18:19:05 2011 +0100

    lirc: build fix
    (cherry picked from commit 942ab6c31026a84e4c5503cb8e68a5135dfc2f2b)
    
    Signed-off-by: Pierre Ynard <linkfanel@yahoo.fr>

Index: vlc-1.1.6/modules/control/lirc.c
===================================================================
--- vlc-1.1.6.orig/modules/control/lirc.c
+++ vlc-1.1.6/modules/control/lirc.c
@@ -25,6 +25,7 @@
  * Preamble
  *****************************************************************************/
 
+#include <errno.h>
 #include <fcntl.h>
 
 #ifdef HAVE_CONFIG_H
@@ -154,10 +155,12 @@ static void Run( intf_thread_t *p_intf )
         /* Wait for data */
         struct pollfd ufd = { .fd = p_sys->i_fd, .events = POLLIN, .revents = 0 };
         if( poll( &ufd, 1, -1 ) == -1 )
+        {
             if( errno == EINTR )
                 continue;
             else
                 break;
+        }
 
         /* Process */
         int canc = vlc_savecancel();
